pygmt/src/project.py

```diff
@@ -241,7 +236,13 @@ def project(
     if output_type == "pandas" and kwargs.get("G") is not None:
         column_names = list("rsp")
 
-    aliasdict = AliasSystem().add_common(
+    aliasdict = AliasSystem(
+        C=Alias(center, name="center", sep="/", size=2),
+        E=Alias(endpoint, name="endpoint", sep="/", size=2),
+        L=Alias(length, name="length", sep="/", size=2, mapping={"limit": "w"}),
```

Comment:
I've added doctests in PR https://github.com/GenericMappingTools/pygmt/pull/4173 to ensure that parameters `sep` and `mapping` can coexist.